reactor-net/src/main/java/reactor/io/net/impl/netty/tcp/NettyTcpServer.java

```diff
@@ -99,22 +98,16 @@ protected NettyTcpServer(@Nonnull Environment env,
 				.option(ChannelOption.SO_BACKLOG, options.backlog())
 				.option(ChannelOption.SO_RCVBUF, options.rcvbuf())
 				.option(ChannelOption.SO_SNDBUF, options.sndbuf())
+				.option(ChannelOption.SO_KEEPALIVE, options.keepAlive())
 				.option(ChannelOption.SO_REUSEADDR, options.reuseAddr())
+				.option(ChannelOption.SO_LINGER, options.linger())
+				.option(ChannelOption.TCP_NODELAY, options.tcpNoDelay())
 				.localAddress((null == listenAddress ? new InetSocketAddress(3000) : listenAddress))
 				.childOption(ChannelOption.ALLOCATOR, PooledByteBufAllocator.DEFAULT)
 				.childOption(ChannelOption.AUTO_READ, sslOptions != null)
 				.childHandler(new ChannelInitializer<SocketChannel>() {
 					@Override
 					public void initChannel(final SocketChannel ch) throws Exception {
-						SocketChannelConfig config = ch.config();
-						config.setReceiveBufferSize(options.rcvbuf());
-						config.setSendBufferSize(options.sndbuf());
-						config.setKeepAlive(options.keepAlive());
-						config.setReuseAddress(options.reuseAddr());
-						config.setSoLinger(options.linger());
-						config.setTcpNoDelay(options.tcpNoDelay());
```

Comment:
no need to set these again. options will have already been set by Netty. Adding the missing options above.